account_multicurrency_revaluation_type/tests/test_currency_revaluation.py

```diff
@@ -0,0 +1,173 @@
+            'company_ids': [(4, cls.company.id, False)]
+        })
+        cls.env.user.company_id = cls.company
+
+        cls.reval_journal = ref(
+            'account_multicurrency_revaluation.reval_journal')
+
+        sales_journal = ref('account_multicurrency_revaluation.sales_journal')
+
+        receivable_acc = ref(
+            'account_multicurrency_revaluation.demo_acc_receivable')
+        receivable_acc.write({'reconcile': True})
+        payable_acc = ref(
+            'account_multicurrency_revaluation.demo_acc_payable')
+        revenue_acc = ref('account_multicurrency_revaluation.'
+                          'demo_acc_revenue')
+
+        # create invoice in USD
+        usd_currency = ref('base.USD')
+
+        bank_journal_usd = ref(
+            'account_multicurrency_revaluation.bank_journal_usd')
+        bank_journal_usd.currency_id = usd_currency.id
+
+        invoice_line_data = {
+            'product_id': ref('product.product_product_5').id,
+            'quantity': 1.0,
+            'account_id': revenue_acc.id,
+            'name': 'product test 5',
+            'price_unit': 800.00,
+            'currency_id': usd_currency.id
+        }
+
+        partner = ref('base.res_partner_3')
+        partner.company_id = cls.company.id
+        partner.property_account_payable_id = receivable_acc.id
+        partner.property_account_receivable_id = payable_acc.id
+
+        payment_term = ref('account.account_payment_term')
+
+        invoice = cls.env['account.invoice'].create({
+            'name': "Customer Invoice",
+            'date_invoice': cls.jan_31,
+            'currency_id': usd_currency.id,
+            'company_id': cls.company.id,
+            'journal_id': sales_journal.id,
+            'partner_id': partner.id,
+            'account_id': receivable_acc.id,
+            'invoice_line_ids': [(0, 0, invoice_line_data)],
+            'payment_term_id': payment_term.id,
+        })
+        # Validate invoice
+        invoice.action_invoice_open()
+
+        payment_method = ref('account_multicurrency_revaluation.'
+                             'account_payment_method_manual_in')
+
+        # Register partial payment
+        payment = cls.env['account.payment'].create({
+            'invoice_ids': [(4, invoice.id, 0)],
+            'amount': 500,
+            'currency_id': usd_currency.id,
+            'company_id': cls.company.id,
+            'payment_date': cls.jan_31,
+            'communication': 'Invoice partial payment',
+            'partner_id': invoice.partner_id.id,
+            'partner_type': 'customer',
+            'journal_id': bank_journal_usd.id,
+            'payment_type': 'inbound',
+            'payment_method_id': payment_method.id,
+            'payment_difference_handling': 'open',
+            'writeoff_account_id': False,
+        })
+        payment.post()
+
+    def test_revaluation_loss(self):
+        # We test that wizard take right currency rate - month rate instead
```

Comment:
why not used the """ """ for docsting ?

Reply:
comments are drooped docstring no,  it works faster